source/d3d12/d3d12_command_list.cpp

```diff
@@ -6,23 +6,66 @@
 #include "log.hpp"
 #include "d3d12_device.hpp"
 #include "d3d12_command_list.hpp"
+#include <assert.h>
+#include "runtime_d3d12.hpp"
 
-D3D12CommandList::D3D12CommandList(D3D12Device *device, ID3D12GraphicsCommandList *original) :
+D3D12CommandList::D3D12CommandList(D3D12Device *device, UINT nodeMask, D3D12_COMMAND_LIST_TYPE type, ID3D12CommandAllocator *pCommandAllocator, ID3D12PipelineState *pInitialState, ID3D12GraphicsCommandList *original) :
```

Comment:
Would be great if you could just revert these two files to how they are in master.

Reply:
Yes, but i have to update the dx12.h ref. I guess it has somthing to do with this commit: https://github.com/crosire/reshade/commit/40621845a7bea529e336e1013450de96c883256d (Revert all projects back to Windows SDK 10.0.17763.0)

Reply:
I guess i have to revert back to Windows SDK 10.0.18362.0.

Reply:
I already fixed D3D12CommandList to work with 10.0.17763.0, so you shouldn't need to modify it at all. Just merge master into your branch and keep the files as they are and it should work.

Reply:
Ok. Will do it as soon as i can ;-)

Reply:
It's done...